packages/core/src/modules/credentials/CredentialsModule.ts

```diff
@@ -67,6 +67,7 @@ export interface CredentialsModule<CFs extends CredentialFormat[], CSs extends C
 
   // Credential
   acceptCredential(options: AcceptCredentialOptions): Promise<CredentialExchangeRecord>
+  sendProblemReport(credentialRecordId: string, message: string): Promise<CredentialExchangeRecord>
```

Comment:
Maybe we can also make this an object like we do with the other methods that contain both the `credentialRecordId` and `message`? Mainly for two reasons:
1. Consistency with other methods
2. It would allow to add more options in the future without needing to make a breaking change (we'll probably want to make it an options object when adding more options)

Reply:
@TimoGlastra refactored